src/Matrix.test.js

```diff
@@ -85,4 +86,23 @@ describe("Matrix tests", () => {
     }
   });
 
+  test("subtraction of values", async() => {
```

Comment:
Just scrap this test, we only need the one in `calculate.test.js`.  Having this test too was a mistake on the last PR